web/oauth/google.ex

```diff
@@ -28,9 +28,9 @@ defmodule RemoteRetro.OAuth.Google do
 
   defp client do
     @oauth_client.new([
-      client_id: System.get_env("REMOTE_RETRO_GOOGLE_OAUTH_CLIENT_ID"),
-      client_secret: System.get_env("REMOTE_RETRO_GOOGLE_OAUTH_CLIENT_SECRET"),
-      redirect_uri: System.get_env("REMOTE_RETRO_GOOGLE_OAUTH_REDIRECT_URI"),
+      client_id: Application.get_env(:remote_retro, :google_oauth)[:client_id],
```

Comment:
let's extract `Application.get_env(:remote_retro, :google_oauth)` out into a variable to avoid the repetition here